src/vdaf/xof.rs

```diff
@@ -330,6 +350,9 @@ pub struct XofFixedKeyAes128 {
     base_block: Block,
 }
 
+// This impl is ONLY used by Mastic. It does not have to comply with the VDAF spec.
```

Comment:
FWIW, I think Mastic will want to move to the `XofFixedKeyAes128Key` interface eventually, once we turn from implementation to optimization. Computation in IDPF/VIDPF-based protocols is almost all XOF invocations, so we'll definitely want to compute the fixed keys just once.

I wouldn't say that this doesn't have to comply with the VDAF spec -- in fact, we make use of this implementation to run common XOF tests, and then we also have a test (`xof_fixed_key_aes128_alternate_apis()`) to confirm that the `Xof` constructor pathway and the key reuse pathway produce the same results. Rather, I'd think about this implementation as a direct reflection of satisfying the `Xof` interface in the spec, and `XofFixedKeyAes128Key` is an implementation-specific optimization, producing the same results. (and in fact, the spec has an implementation note suggesting caching the fixed key)

Reply:
Makes sense. I've toned down the comment a bit. Agreed that having a unified API would be nice. It'll be a little bit of a lift